index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>aboutLori</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+
+<link rel="stylesheet" href="style.css"> 
+
+<body>
+                  
+<!-- TOP TRANSPARENT NAV BAR -->
+    <div class="topMenu">
+        <img id="profilepic"
+            src="images/headshot.jpg"
+             alt="profilePic"> 
+
+    <div class="title">
+                    <h1> Lori Stravers MPH, CCRC, CHES</h1> 
+    </div>
+        
+        <nav class="navigationMenu">
+            <ul>
+                <a href="https://github.com/lstravers" target="_blank">
+                <i class="fa fa-github" style="font-size:36px; color: black"></i>
```

Comment:
Instead of using an inline style, you can add another class to this tag, even though it came from Font Awesome, and then add the styles to that class.